owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/settings/SettingsSecurityViewModelTest.kt

```diff
@@ -38,11 +40,13 @@ import org.junit.Test
 class SettingsSecurityViewModelTest : ViewModelTest() {
     private lateinit var securityViewModel: SettingsSecurityViewModel
     private lateinit var preferencesProvider: SharedPreferencesProvider
+    private lateinit var contextProvider: ContextProvider
 
     @Before
     fun setUp() {
         preferencesProvider = mockk(relaxUnitFun = true)
-        securityViewModel = SettingsSecurityViewModel(preferencesProvider)
+        contextProvider = mockk()
```

Comment:
Maybe in this case it is not necessary, but just a tip: when you mock an object with MockK, it's usually very helpful (and sometimes it avoids headaches) to pass `relaxUnitFun = true` as parameter. This mocks automatically the methods of the object which return Unit (which sometimes are called internally, not directly by us in our code), preventing you of having to mock all of them in the same way 😀

Reply:
Okey, thanks!